tests/conftest.py

```diff
@@ -548,7 +548,9 @@ def full_record(users):
             ],
             "references": [
                 {
-                    "reference": "0000 0001 1456 7559",
+                    "reference": "Nielsen et al,..",
+                    "identifier": "0000 0001 1456 7559",
+                    "scheme": "isni",
```

Comment:
Remark: this fixture was not affecting the MARCXML serializer test, but I changed it anyway so we're following the same format everywhere.

Reply:
OK nice.
What I just did is to remove the change to `full_record` and only keep the changes in `serializers/conftest.py`.
This way, you should be able to merge this PR and not have conflicts in the performance improvements branch.

Reply:
Sorry I was wrong, `full_record` is what is used in `test_marcxml_serializer_full_record`.
I'll mark this pull request as draft for now, and come back to it once you're done with the other more important pull requests.

Reply:
I confirm that following @alejandromumo's latest changes, the test in `test_marcxml_serializer.py` is not using `full_record_to_dict`. So now, the first comment I wrote is correct:

> Remark: this fixture was not affecting the MARCXML serializer test, but I changed it anyway so we're following the same format everywhere.